core/src/test/scala/tech/beshu/ror/unit/acl/factory/decoders/definitions/LdapServicesSettingsTests.scala

```diff
@@ -47,9 +47,22 @@ class LdapServicesSettingsTests(ldapConnectionPoolProvider: UnboundidLdapConnect
 
   private val containerLdap1 = new LdapContainer("LDAP1", "test_example.ldif")
   private val containerLdap2 = new LdapContainer("LDAP2", "test_example.ldif")
+  private var dnsContainer: Option[DnsServerContainer] = None
```

Comment:
same here - why not to move it to MultipleContainters?

Reply:
as above